test/functional/change_stream.test.js

```diff
@@ -54,9 +59,12 @@ function tryNext(changeStream, callback) {
   let complete = false;
   function done(err, result) {
     if (complete) return;
-
     // if the arity is 1 then this a callback for `more`
     if (arguments.length === 1) {
+      if (err instanceof Error) {
```

Comment:
in what case is an error returned here? When I look at where `more` is emitted in `change_stream.js`, the errors are always handled. This would seem to imply the `getMore` callback was called with `err, err` or something.

Reply:
Oops I think I should remove this now, I was just trying to track down an issue but it's no longer necessary.